src/analysis/rule_extraction.jl

```diff
@@ -129,8 +138,10 @@ A StableRules object (implemented by SIRUS).
    Electron. J. Statist. 15 (1) 427 - 505.
    https://doi.org//10.1214/20-EJS1792
 """
-function cluster_rules(clusters::Vector{T}, X::DataFrame, max_rules::T;
-    seed::Int64=123, kwargs...) where {T<:Int64}
+function cluster_rules(
+    clusters::Vector{T}, X::DataFrame, max_rules::T;
+    seed::Int64=123, remove_duplicates::Bool=true, kwargs...
+)::Vector{Rule{Vector{Vector},Vector{Float64}}} where {T<:Int64}
```

Comment:
Was this the formatter?

Reply:
No, it was me, because this is in agreement with Blue Style Guide (but I'm happy to change it)

Reply:
I don't see it:

![image](https://github.com/user-attachments/assets/99b14627-1d3e-4a91-98a8-6a1b0511b6be)

![image](https://github.com/user-attachments/assets/d18e1e08-1982-4191-9353-00b16b842cc2)

I'm looking at the README in the github project, maybe you're referring to an older version?

https://github.com/JuliaDiff/BlueStyle?tab=readme-ov-file#method-definitions

Reply:
The second example here.

![image](https://github.com/user-attachments/assets/9e831031-a3ac-4ac1-a207-12a43e12572b)

Reply:
Okay, fine with me!